test/dashboard-e2e/config.ts

```diff
@@ -12,7 +10,9 @@ const config = {
   apiUrl: process.env.API_URL!,
   dashboardApiUrl: process.env.DASHBOARD_API_URL!,
 
-  runId: process.env.RUN_ID || randomUUID(),
+  runId: process.env.RUN_ID,
```

Comment:
Should it actually have different ID for every single resource when `RUN_ID` is not passed?

Reply:
The goal was to make resources unique for every execution, so it's fine to have different ids when specific `RUN_ID` isn't defined. Even with RUN_ID specified it may differ across resources - max length for k8s resource names is 63 characters - the ID may be trimmed.

Reply:
`randomUUID().replace(/-/g, '')` then :)